src/commands/essentials/permissionset/generate.ts

```diff
@@ -329,6 +329,22 @@ export default class PermissionSetGenerate extends Command {
     // Process filtering and additional elements
     public filterPackageXmlTypes(packageXmlTypes: any, packageXMLTypesAll: any, permissionSetDefinition: any) {
         let permissionSetsXmlElement = '';
+
+        let packageXmlTypesArray = [];
+        for (const packageXmlTypeElement of packageXmlTypes) {
+            packageXmlTypesArray.push(packageXmlTypeElement.name[0]);
+        }
+        // Manage elements that are in permissionSetDefinition JSON and not in packageXmlTypes
+        for (const permissionSetElement of permissionSetDefinition.packageXMLTypeList) {
+            if (!packageXmlTypesArray.includes(permissionSetElement.typeName) && packageXMLTypesAll.includes(permissionSetElement.typeName) ) {
```

Comment:
That's why you have a member with (.*) in the generate PS xml file  ...
It's possible that the solution is not to add virtual stuff in packageXmlTypes, but to add the element in packageXmlTypesArray

Reply:
Add check on includedFilterList and additionalElements